core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3521,17 +3521,17 @@ private void _cleanUpShutdownTimer(List<Throwable> errors) {
 
     private void _cleanUpShutdownTcpSlaveAgent(List<Throwable> errors) {
         if(tcpSlaveAgentListener!=null) {
-            LOGGER.log(FINE, "Shutting down TCP/IP slave agent listener");
+            LOGGER.log(FINE, "Shutting down TCP/IP agent listener");
```

Comment:
I would really suggest switching to "client" here. Not only agents may use this thread for connection